packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -516,4 +516,60 @@ describe('masternode', () => {
       expect(data.length).toBe(64)
     })
   })
+
+  describe('accountHistoryCount', () => {
+    it('should get accountHistoryCount', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.historyCount()
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with owner as all', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.historyCount('all')
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with no_rewards option', async () => {
+      await waitForExpect(async () => {
+        const options: AccountHistoryCountOptions = {
+          no_rewards: true
+        }
+        const count = await client.account.historyCount('mine', options)
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with token option', async () => {
+      await waitForExpect(async () => {
+        const options: AccountHistoryCountOptions = {
+          token: 'DBTC'
+        }
+        const count = await client.account.historyCount('mine', options)
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
```

Comment:
this test you can actually test 'DBTC', 'DETH' and 'DFI' which will output different count, other than `toBeGreaterThanOrEqual`, you can expect the result in exact count. 
you can refer to the fixture `setup` above

Reply:
Did not know that. let me try doing that.